src/els_dap_general_provider.erl

```diff
@@ -316,3 +351,25 @@ frame_by_id(FrameId, Threads) ->
 -spec format_mfa(module(), atom(), integer()) -> binary().
 format_mfa(M, F, A) ->
   els_utils:to_binary(io_lib:format("~p:~p/~p", [M, F, A])).
+
+-spec parse_mfa(string()) -> {module(), atom(), non_neg_integer()} | error.
+parse_mfa(MFABinary) ->
+  MFA = unicode:characters_to_list(MFABinary),
+  ?LOG_DEBUG("tokens for ~p: ~w", [MFA,  erl_scan:string(MFA)]),
```

Comment:
Nit: we could store the output of `erl_scan:string(MFA)` into a variable, instead of performing the operation twice.

Reply:
That was leftover debug print, let me remove these two lines